app/src/main/java/com/andrew/tasky/agenda/data/storage/InternalStorage.kt

```diff
@@ -0,0 +1,30 @@
+package com.andrew.tasky.agenda.data.storage
+
+import android.content.Context
+import com.andrew.tasky.agenda.data.event.photo.LocalPhotoDto
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import okio.IOException
+
+class InternalStorage(private val context: Context) {
```

Comment:
You'll also need a function to delete an image after it has been uploaded to the server